src/sync/request_id.rs

```diff
@@ -16,9 +16,9 @@ lazy_static! {
     static ref SYNC_COUNTERS: Mutex<HashMap<String, AtomicU32>> = Mutex::new(HashMap::new());
 }
 
-// new() returns a new syncid of the form <clientid>-<sessionid>-<sync count>.
+// new() returns a new request_id of the form <clientid>-<sessionid>-<sync count>.
 // The sync count enables one to find the sync following or preceeding a given
```

Comment:
Here in lines 19-41 we should probably stop referring to them as "sync count" and "SYNC_COUNTER" and call it request counter.